packages/rich-text/src/plugins/EmbeddedEntityInline/FetchingWrappedInlineEntryCard.tsx

```diff
@@ -28,25 +32,26 @@ interface FetchingWrappedInlineEntryCardProps {
 }
 
 export function FetchingWrappedInlineEntryCard(props: FetchingWrappedInlineEntryCardProps) {
-  const { getEntry, loadEntityScheduledActions, entries } = useEntities();
-  const entry = React.useMemo(() => entries[props.entryId], [entries, props.entryId]);
+  const { data: entry, status: requestStatus } = useEntity<Entry>('Entry', props.entryId);
+  const { getEntityScheduledActions } = useEntityLoader();
+  const loadEntityScheduledActions = () => getEntityScheduledActions('Entry', props.entryId);
 
   const allContentTypes = props.sdk.space.getCachedContentTypes();
   const { onEntityFetchComplete } = props;
   const contentType = React.useMemo(() => {
-    if (!entry || entry === 'failed' || !allContentTypes) return undefined;
+    if (!entry || !allContentTypes) return undefined;
```

Comment:
Should we also add a condition for `requestStatus !== 'success'` here?

Reply:
Previously if the entry would equal the string `failed` if there was an error. Now if there is an error it returns undefined so I don't think that would be necessary